### DISCUSSION TITLE
Welcome to FG-Falcon Discussions!

### DISCUSSION THREAD

**Discussion by jakka351** [jakka351/FG-Falcon#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by animation303** [comment]
Hello there, my name is Tunsafun

My intention is to create a SBC based head unit using the resources found here, I own a 2012 Ford Territory base model, and purchased an android based head unit replacement and was extremely unhappy with it, so I decided I'm going to make my own.

**Comment by sreid81** [comment]
First of all I love all the hard yards you've put into the fg falcon , good on ya
Are you still doing the openicc project, I'm currently running auto open pro at the moment but I'm not happy with it, you're openicc is definitely going down the path I was aiming for.
I would be up for paying for the software as well and I bet alot of others would be too
